[Magic Firewall] Added About and updated landing page (#2980)
cloudflare/cloudflare-docs: products/magic-firewall/src/content/about/list-types.md

---
title: List types
pcx-content-type: concept
---

# List types

## Threat intelligence

> Cloudflare sees approximately 28 million HTTP requests each second and blocks 76 billion cyber threats each day. Cloudflare uses that data to detect malicious actors on the Internet and turns that information into a list of known malicious IPs. Cloudflare also integrates with a number of third party vendors to augment the coverage.

Reviewer: ```suggestion
Cloudflare sees approximately 28 million HTTP requests each second and blocks 76 billion cyber threats each day. Cloudflare uses that data to detect malicious actors on the Internet and turns that information into a list of known malicious IP addresses. Cloudflare also integrates with a number of third-party vendors to augment the coverage.
```

> The threat intelligence feed categories include Malware, Anonymizer, and Botnet Command-and-Control centers. Malware and Botnet lists cover properties on the Internet distributing malware and known command and control centers. Anonymizers contain a list of known forward proxies that allow attackers to hide their IP addresses.

Reviewer: ```suggestion
The threat intelligence feed categories include Malware, Anonymizer, and Botnet Command-and-Control centers. Malware and Botnet lists cover properties on the Internet distributing malware and known command-and-control centers. Anonymizers contain a list of known forward proxies that allow attackers to hide their IP addresses.
```

## IP lists

> IP lists can be used to easily group services in networks, like webservers, or for lists of known bad IPs to make managing good network endpoints easier. IP lists are helpful for users with very expansive firewall rules with many IP lists, and users can add up to 100,000 IPs per list that can used in rules.

Reviewer: ```suggestion
IP lists can be used to easily group services in networks, like web servers, or for lists of known bad IP addresses to make managing good network endpoints easier. IP lists are helpful for users with very expansive Firewall Rules with many IP lists. You can add up to 100,000 IPs per list.
```
FYI: I believe I capitalized "IP Lists" in the Firewall Rules docs, but we can agree on not doing so. :)

Author: Fixed!
